src/admin/class-admin.php

```diff
@@ -21,6 +20,26 @@
  */
 class Admin implements Service {
 
+  /**
+   * Manifest Data Interface.
+   *
+   * @var Manifest_Data
```

Comment:
```suggestion
   * @var object
```

Explanation is down, as I totally missed this 😆

Reply:
👍